swoloszynski/journey-planner
# Issue 6 add sql db, closes #6

### What
- Instructions and configs for setting up the local database

### How to test
- Follow instructions and confirm that you can init a local database and run these migrations
